src/main/resources/static/js/login.js

```diff
@@ -1,13 +1,81 @@
 
 jQuery(document).ready(function() {
-	
-//	document.getElementById("loginBtn").onclick = function() {		  
-//		
-//    	var iframe = document.getElementById("mainFrame");
-//    	iframe.src = "http://localhost:8080/subscriptionpage.html";
-//	}
-	
-
-	
+
+    // Fetch injected URL from DOM
+    frontendServiceUrl = $('#frontendServiceUrl').text();
+
+    // /Start ## Global AJAX Sender function ##################################
+    var AjaxHttpSender = function () {};
+
+    AjaxHttpSender.prototype.sendAjax = function (url, type, data, callback) {
+        $.ajax({
+            url : url,
+            type : type,
+            data : data,
+            contentType : 'application/json; charset=utf-8',
+            dataType : "json",
+            cache: false,
+            beforeSend : function () {
+                callback.beforeSend();
+            },
+            error : function (XMLHttpRequest, textStatus, errorThrown) {
+                callback.error(XMLHttpRequest, errorThrown);
+            },
+            success : function (data, textStatus) {
+
+                callback.success();
+            },
+            complete : function (XMLHttpRequest, textStatus) {
+                callback.complete();
+            }
+        });
+    }
+    // /Stop ## Global AJAX Sender function ##################################
+
+    // /Start ## Knockout ####################################################
+    function loginModel() {
+        this.userState = {
+            username: ko.observable(""),
+            password: ko.observable(""),
+            remember: ko.observable(false)
```

Comment:
i did not found **remember** related functionality.

Reply:
When user check this checkbox, the user name and password will store in cookies,
If user tries to login the cookies details are automatically populates into UserName and Password field.

Is there any other functionality for this remember? what is the use to store this checkbox value?

Reply:
oh. as I understand now, it is not needed to pass remember value to backend, but add this functionality to frontend.. I will fix it. thanks